client/lib/analytics/super-props.js

```diff
@@ -29,6 +31,7 @@ const getSuperProps = ( reduxStore ) => ( eventProperties ) => {
 
 			site_id_label: selectedSite.jetpack ? 'jetpack' : 'wpcom',
 			site_plan_id: selectedSite.plan ? selectedSite.plan.product_id : null,
+			site: siteSlug,
```

Comment:
Hi @jsnajdr, I made a change in `super-props` to track additional data, and I wasn't sure who would be the best person to review this change. Since you've worked on analytics changes in this file in the past, thought I'd ping you to make sure I'm not breaking anything with this change!